common_dwp_api/src/test/java/uk/gov/dhsc/htbhf/dwp/model/DWPEligibilityRequestTest.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.dhsc.htbhf.dwp.model;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.dhsc.htbhf.assertions.AbstractValidationTest;
+
+import java.util.Set;
+import javax.validation.ConstraintViolation;
+
+import static uk.gov.dhsc.htbhf.assertions.ConstraintViolationAssert.assertThat;
+import static uk.gov.dhsc.htbhf.dwp.testhelper.DWPEligibilityRequestTestDataFactory.aValidDWPEligibilityRequestV2;
+import static uk.gov.dhsc.htbhf.dwp.testhelper.DWPEligibilityRequestTestDataFactory.aValidDWPEligibilityRequestV2WithEligibilityEndDate;
+import static uk.gov.dhsc.htbhf.dwp.testhelper.DWPEligibilityRequestTestDataFactory.aValidDWPEligibilityRequestV2WithPerson;
+import static uk.gov.dhsc.htbhf.dwp.testhelper.PersonDTOTestDataFactory.aPersonDTOWithSurname;
+
+class DWPEligibilityRequestTest extends AbstractValidationTest {
```

Comment:
Combining a  rename and a package move seems to have caused this class to appear as a new file.